dist/tools/pr_check/no_merge_keywords

```diff
@@ -0,0 +1,12 @@
+^    [0-9a-f]\+ \<SQUASH\>
+^    [0-9a-f]\+ FIX
+^    [0-9a-f]\+ \<REMOVE *ME\>
+^    [0-9a-f]\+ \<Update\>
```

Comment:
At least for the legacy keywords, I would prefer to not introduce word boundaries.